examples/cms-starter/datasources/categories.json

```diff
@@ -0,0 +1,429 @@
+{
+    "id": "categories",
+    "fields": {
+        "Title": {
+            "type": "string"
+        },
+        "Color": {
+            "type": "color"
+        }
+    },
+    "items": [
+        {
+            "Title": "CMS",
+            "Color": "orange"
+        },
+        {
+            "Title": "Basics",
+            "Color": "red"
+        },
+        {
+            "Title": "Updates",
+            "Color": "blue"
+        },
+        {
+            "Title": "Pro Tips",
+            "Color": "green"
+        },
+        {
+            "Title": "Category 1",
```

Comment:
Is there any need to have these numbered categories? Are the above ones not sufficient?

Reply:
No! Let's clean this up 👀